core/storage/classifier/gae_models.py

```diff
@@ -181,6 +181,25 @@ def create(
         training_job_instance.put()
         return instance_id
 
+    @classmethod
+    def query_training_jobs(cls, cursor=None):
+        """Generates a unique id for the training job of the form
```

Comment:
I am sorry I thought I added the changes in this file. The changes should show up in my next push along with next_scheduled_check_time changes.